package/firmware/linux-firmware/intel.mk

```diff
@@ -7,6 +7,13 @@ define Package/ibt-firmware/install
 endef
 $(eval $(call BuildPackage,ibt-firmware))
 
+Package/iwlcc-a0-firmware = $(call Package/firmware-default,Intel AX200 firmware)
```

Comment:
Could you please follow the naming schema used for the other iwl firmware files like `iwlwifi-firmware-ax200`

Reply:
HI Hauke,
Thank your advice.
I already change package name from iwlcc-a0-firmware to iwlwifi-firmware-ax200.
Thank you.